semver.py

```diff
@@ -83,7 +82,26 @@ class VersionInfo(collections.namedtuple(
     :param str prerelease: an optional prerelease string
     :param str build: an optional build string
     """
-    __slots__ = ()
```

Comment:
So now VersionInfo is a mutable object with dynamic attrs. That's quite too dramatic change in the name of pretty printting pandas dataframes /:

Reply:
Nothing else than Pandas printing but that's quite problematic as it won't be solved soon (my 2 cts)

Some tests to ensure that VersionInfo are immutable should be added if mutability of VersionInfo is a problem